metricbeat/module/kubernetes/util/kubernetes.go

```diff
@@ -574,3 +574,24 @@ func GetClusterECSMeta(cfg *conf.C, client k8sclient.Interface, logger *logp.Log
 	}
 	return ecsClusterMeta, nil
 }
+
+// AddClusterECSMeta adds ECS orchestrator fields
+func AddClusterECSMeta(base mb.BaseMetricSet) mapstr.M {
```

Comment:
Can this be also used at https://github.com/elastic/beats/blob/main/metricbeat/module/kubernetes/event/event.go#L102